src/apps/relay/mainrelay.c

```diff
@@ -2622,19 +2615,7 @@ int main(int argc, char **argv)
 	event_add(ev, NULL);
 
 	drop_privileges();
-#if !defined(TURN_NO_PROMETHEUS)
-	int prometheus_status = start_prometheus_server();
-	if (prometheus_status < 0) {
-	  TURN_LOG_FUNC(TURN_LOG_LEVEL_ERROR, "Could not start Prometheus collector!\n");
-	}
-	else if (prometheus_status == 1) {
-	  TURN_LOG_FUNC(TURN_LOG_LEVEL_INFO, "Prometheus collector disabled, not started.\n");
-	}
-	else {
-	  TURN_LOG_FUNC(TURN_LOG_LEVEL_INFO, "Prometheus collector started successfully.\n");
-	}
-#endif
-
+	start_prometheus_server();
```

Comment:
Will figure out internally if prometheus is supporter and enabled
In other cases do nothing